src/providers/webhook.js

```diff
@@ -0,0 +1,19 @@
+export default (options) => {
+  return {
+    id: "webhook",
+    type: "webhook",
+    name: "Webhook",
+    maxAge: 24 * 60 * 60, // How long link are valid for (default 24h)
+    ...options,
+    sendVerificationRequest: ({
```

Comment:
The name is "inherited" from the `email` provider, nothing says it couldn't change.